benches/texts/ar/multiple_paragraphs.txt

```diff
@@ -0,0 +1,29 @@
+الشركة نفسها مهمة جدًا، لكن الدليل مهمل لسببين. والبعض اختار اثنين. متى تم اعلان النتائج من يجب أن يستخدم شيئًا معي، خطأي لكل واحد. فليكن خيرًا للسهلين، حتى لا يتخلف العزل.
+
+لإسقاط إرادتي وقمعي. لا نريد أن نزعجه بهذا القرار. لأنه أوضح لها أن الجارية لم تتعلم منها، وعادة ما يوضح ذلك لقد أنقذته قوة المجهول، وهي نتيجة قوتي. علاوة على ذلك، فإن مشكلة الأبيقوري في البداية، إذا أراد أن يتخلص منها، فلن أتفق معه إذا لم يتخلوا عنها. يتباهى أحدهم بأنه ولد في كثير من الأحيان بأداء لم يكن موجودا.
+
+وأنزل حق الفصاحة بلا فساد، فإن الواجبات لا ينبغي طلبها للتعريف، فهي في الحقيقة تسمى. أي مكتب في أي وقت يتم من خلاله الدفع له ولهؤلاء الأفراد. عندما تنتهي الجودة ولا، يكون لدينا الجزء الذي تم حله، إيروس أبيقور وهؤلاء. البحر للتفكير في المخاطر. بيرسيوس نحن نأمر لا، ألم الكياسة يا عزيزي لا. أو فقط لجعل رجل حكيم، ولا تريتاني، ويقول بحق، في نفس الوقت سيساعدهم على ذلك
+
+في الغابة، ملذات القوة الحالية موجودة؛ لإزالة ترتيب هذه. عادة ما يكون مجرد واجب منزلي. ولكن مع المشجعين وما شابه ذلك. فهو موجود على أعلى مستوى، ونحن، حتى لو قلت للمواطنين، لا نتهمه بشكل فضفاض وعن حق.
+
+سأدعك تتركه فحسب، وسيستقر عليه استخدام الدعاية في الوقت المناسب. اعتقد القدماء أن العسل لا ينبغي أن يكون مفيدا. بالنسبة لهم، سيكون لدينا لهم. ليس لدي عدو ولا قوة، هل هذا لطيف بالنسبة لتريتان، أم أن سعادتي تبدو للآخرين؟
+
+أقوم بسحب ألبوسير الخاص بي حتى أفهم أنه لا يخطر ببالي أنه موجود على الإنترنت. أنه يجب أن يكون أول من يعمل، وأن يفهم أنه كان لئيمًا معها، تجاه أنتوب. أو تلك المتع الأخرى. إنهم خائفون من الغابة ويخافون. يبدو من غير المعتاد رفض كرتي قدم.
+
+بالكاد ساعد مع البظر. في الخرافات الأولى تعريفات العسل يقولون أنه يجب أن نحتج على هؤلاء بسبب آلامي. شاب ذو عقل، أو إذا كنت مدينًا لهم بالتوبيخ، سيختلف مع الحق العنيد. يبدو أنك تمتلكه، إنه أمر مميز بالنسبة له. سأبذل قصارى جهدي للقيام بذلك، فهي الطريقة الوحيدة لتعليمهم.
+
+لكن قوتي كانت جاهزة للفتح. اليوناني الأول الذي قرأنا فيه هو. عندما يكون الأداء متوسطًا أو الفشل، فإن هذه القوة سوف تقطعك. كما فهمي الكبير. دعوهم يكرهوننا. يقولون إنهم مخطئون، ويختلفون معه، أنه إذا أراد أن يكون بكل قوته، فإنه يخشى أن يعيش ويأخذ على عاتقه. لا يوجد سبب له.
+
+لا ينبغي لقوة صمغنا أن تسيء إلينا أو تعوق تلك الأعمال. يحسبونك ثوب أجنبي يا عسل كذبة متاعب وكل يوم معهم. ولا في أي وظيفة أنت تاسع عنادي السهل. لقد هجرك الرجال في البداية، عندما تم الازدراء بمثل هذه الأقوال. لأنه لم يتم إثبات أي خطأ، ونأمل أن يكون ذلك بالنسبة إلى ديموقريطوس الحكيم اللاحق. لا تكرهه. كميزة معينة لا.
+
+سأفتح له ملحي الصغير بالعسل في بلاط البلاد. وتم حل بعضها. في الصراع الأبدي الكبير على السلطة، والذي يكون سهلًا دائمًا في هوميروس، أمام ذلك الأجنبي. السؤال هو هل هو رجل حكيم أم بطل أعلى؟
+
+الاثنان ليسا مجرد أوزان، لكنهما ليسا رقيقين على الإطلاق. رجال إيروس كذلك. العسل لا يقول إلا الجثث. من أحكامي يُدرك أن الأخطاء لا ترتكب من قبل
+
+اجمعهم مع mnesarch، وهذا ما يقولون أننا نرسل من أجله. يجب أن يكونوا مدينين، إذ رأى، على أمل الجميع، أولئك الذين طلبوا ذلك أولاً. من هذه التوجيهات ullamcorper، تترجم له العسل الذي تم تخفيفه. لوريت متوسط ​​أم لا.
```

Comment:
I think the samples should avoid mixing code points from different scripts and/or directions. Because harfbuzz (and rustybuzz) generally expects the input string after bidi segmentation. (in Arabic, this applies for numbers as well, amusingly enough.)

Reply:
I'll try to make sure of that in the new examples.